openmicroscopy/ome-help
# Updating pages and menu for 5.0.7 release

Update version number to 5.0.7 in Home page, Getting Started 5, Importing Data 5, Training Materials, Previous and CLS pages.

To test check version numbers in:
- menu items
- page titles and headers
- Getting Started Download and Zip screenshots - should show 2 Mac OS X versions
- check note text below Downloads screenshot
- body text in Home page, Training Materials and Previous pages
- /cls-getting-started.html - check screenshots and note below Downloads as above 

Note that the PDFs will not be updated as these are not on staging yet.

15-02-04 - PDFs added to staging.downloads
